docs/plugins.md

```diff
@@ -7,31 +7,36 @@ title: Plugins (Beta)
 
 > The plugin API is in a **beta** state as of Prettier 1.10 and the API may change in the next release!
 
-Plugins are ways of adding new languages to Prettier. Prettier's own implementations of all languages are expressed using the plugin API. The core `prettier` package contains JavaScript and other web-focussed languages built in. For additional languages you'll need to install a plugin.
+Plugins are ways of adding new languages to Prettier. Prettier's own implementations of all languages are expressed using the plugin API. The core `prettier` package contains JavaScript and other web-focused languages built in. For additional languages you'll need to install a plugin.
 
 ## Using Plugins
 
-Plugins are automatically loaded if you have them installed in your `package.json`. Prettier plugin package names must start with `@prettier/plugin-` or `prettier-plugin-` to be registered.
+Plugins are automatically loaded if you have them installed in the same `node_modules` directory where `prettier` is located. Plugin package names must start with `@prettier/plugin-` or `prettier-plugin-` to be registered.
 
-If the plugin is unable to be found automatically, you can load them with:
+When plugins cannot be found automatically, you can load them with:
 
-* The [CLI](./cli.md), via the `--plugin` flag:
+* The [CLI](./cli.md), via the `--plugin` and `--plugin-search-dir`:
```

Comment:
I'm not sure why someone can't use `--plugin` and `--plugin-search-dir` simultaneously – that's technically possible and can be useful when debugging how a bunch of plugins works together (e.g. we have python and php plugins installed globally and we're testing if a locally developed elm plugin with a locally installed prettier does not conflict with the other two). Seeing both options together looks logical to me after this phrase:

> When plugins cannot be found automatically, you can load them with:
 
I don't feel there is much distinction between `--plugin` and `--plugin-search-dir` for a user, so I don't see much value in splitting the narrative into two separate sections. The only potential benefit that I see is consistency with how other CLI options are documented 🤔 

Shall we keep the docs as is until the plugin ecosystem graduates from beta and thus deserves extra attention anyway?

Reply:
I originally meant they could be used simultaneously but not required since they are completely different and won't affect each other.

After second thought, the examples are relative paths so people should understand that they're separated. This PR should be good to go now.